src/persistence/SubjectDatabaseEntityLoader.ts

```diff
@@ -64,7 +64,7 @@ export class SubjectDatabaseEntityLoader {
 
                     // gets all relation property paths that exist in the persisted entity.
                     subject.metadata.relations.forEach(relation => {
-                        const value = relation.getEntityValue(subject.entityWithFulfilledIds!);
+                        const value = relation.getEntityValue(subject.entity!);
```

Comment:
uhh wait wait those places are complex and thus sensitive to change, can you explain your change? Its scary to me to merge changes in this area.